src/runtime_src/core/common/drv/include/xrt_xclbin.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * A GEM style device manager for PCIe based OpenCL accelerators.
+ *
+ * Copyright (C) 2019-2020 Xilinx, Inc. All rights reserved.
```

Comment:
This is a new file. Please remove "2019-". Could we use SPDX-License-Identifier? See kds_core.c